api/api/views/oauth2_views.py

```diff
@@ -136,9 +137,9 @@ def get(self, request, code, format=None):
 
 
 @extend_schema(tags=["auth"])
-class TokenView(BaseTokenView, APIView):
+class TokenView(APIView, BaseTokenView):
     @token
-    def post(self, *args, **kwargs):
+    def post(self, request, *args, **kwargs):
```

Comment:
Looks like after `request` the arguments aren't needed anyomre.
```suggestion
    def post(self, request):
```

Reply:
Sure, I thought it was best to keep the `*args` and `**kwargs` params when overridding methods but they are truly not needed in this case.